Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.6
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install default-jdk -y
+
+WORKDIR /Lino-Webcrawler-Matricula
+
+COPY ./requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Seria bom mudar o COPY para que fique mais legível. Ideia: COPY . /Lino-Webcrawler-Matricula. Mas como vou mexer com o deploy, pode deixar que eu mudo quando for fazer